examples/lighting-app/nrfconnect/main/include/Rpc.h

```diff
@@ -0,0 +1,66 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <kernel.h>
+
+#ifdef CONFIG_CHIP_PW_RPC
+#pragma message "Building with RPC support"
```

Comment:
IMO build log should only contain fatal diagnostics; we don't want to dilute those out with notices. So I'd ask to remove this.

I don't remember how much output a successful nRF Connect build produces but for CHIP and all of the GN examples, there is no non-fatal output at all (except on Mac, but we should fix that!).

Reply:
reason I added this message pragma is that there are basically two build flavors of this example app now: one with Pigweed RPCs and one without. It may be useful to be able to grep/search for this string in the build output to make sure you are building the right flavor.

Reply:
The number of possible variants is more than 2, and far too many to print (it's the whole config.h)

Suggest developers inspect their config.h if interested in what configuration choices were made.

Reply:
done